src/locale/locales/de/messages.po

```diff
@@ -692,68 +654,68 @@ msgstr "Suchanfrage löschen"
 
 #: src/view/screens/Support.tsx:40
 msgid "click here"
-msgstr ""
+msgstr "hier klicken"
 
 #: src/screens/Onboarding/index.tsx:35
 msgid "Climate"
-msgstr ""
+msgstr "Klima"
 
 #: src/view/com/modals/ChangePassword.tsx:265
 #: src/view/com/modals/ChangePassword.tsx:268
 msgid "Close"
-msgstr ""
+msgstr "Schließen"
 
 #: src/components/Dialog/index.web.tsx:78
 msgid "Close active dialog"
-msgstr ""
+msgstr "Aktiven Dialog schließen"
 
 #: src/view/com/auth/login/PasswordUpdatedForm.tsx:38
 msgid "Close alert"
-msgstr ""
+msgstr "Alarm schließen"
 
 #: src/view/com/util/BottomSheetCustomBackdrop.tsx:33
 msgid "Close bottom drawer"
-msgstr ""
+msgstr "Untere Schublade schließen"
 
 #: src/view/com/lightbox/ImageViewing/components/ImageDefaultHeader.tsx:26
 msgid "Close image"
 msgstr "Bild schließen"
 
 #: src/view/com/lightbox/Lightbox.web.tsx:119
 msgid "Close image viewer"
-msgstr ""
+msgstr "Bildbetrachter schließen"
 
 #: src/view/shell/index.web.tsx:49
 msgid "Close navigation footer"
-msgstr ""
+msgstr "Fußzeile der Navigation schließen"
 
 #: src/view/shell/index.web.tsx:50
 msgid "Closes bottom navigation bar"
-msgstr ""
+msgstr "Schließt die untere Navigationsleiste"
 
 #: src/view/com/auth/login/PasswordUpdatedForm.tsx:39
 msgid "Closes password update alert"
-msgstr ""
+msgstr "Schließt die Kennwortaktualisierungsmeldung"
 
 #: src/view/com/composer/Composer.tsx:302
 msgid "Closes post composer and discards post draft"
-msgstr ""
+msgstr "Schließt den Post Composer und verwirft den Beitragsentwurf"
```

Comment:
Is "post composer" a specific tool? Does it mean something like "post aggregation" or "combination"? In that case, we could fine a translation for that.

Later you say "Beitragsentwurf", so again "post" is translated with "Beitrag" (which is fine). But we probably should find a common translation across the entire app.

Reply:
"Post composer" is referring to the modal/dialog box (never quite sure what the right terminology is) that opens when you are typing out a post. So would Beitrag Composer work here perhaps? I agree it's best to stick to a common translation throughout the app.

Reply:
"Beitragsverfasser" would work?

"composer" is not something we have in German.

Reply:
```suggestion
msgstr "Schließt den Beitragsverfasser und verwirft den Beitragsentwurf"
```
How does this look?